src/shared/models/UserModel.js

```diff
@@ -2,4 +2,12 @@ export class UserModel {
     email;
     name;
     nickname;
+
+    static fromBackend(data) {
```

Comment:
I think that a better name for this argument is `user` not the over generic `data`

Reply:
Thanks for catching this! Fixed in [https://github.com/warsawjs/topics-manager/pull/23/commits/2b170b841919de548f8e2da932d2746aec19b448]